### ISSUE TITLE
cgit.kde.org

### ISSUE THREAD

**Issue by a17r** [dragotin/kraft#3]
kraft kf5 branch on cgit.kde.org is now rather far behind. Do you plan to push the changes back there at some point?

**Comment by dragotin** [comment]
Well, I should do, even though I love working on github.

**Comment by a17r** [comment]
git works the same, no? ;)

**Comment by dragotin** [comment]
Kraft was removed from the KDE git. See https://dragotin.wordpress.com/2018/03/22/kraft-out-of-kde/ for more details on how that happened.

**Comment by a17r** [comment]
Well that's unfortunate. Note that I made my PR only here because it would have been pointless to do on KDE infrastructure while you are working on GitHub. I found out only looking for signs of a fork as part of my packaging work. I don't see a workflow difference with git whether it is being hosted on KDE infrastructure or GitHub.

Working on a repository clone and keeping an old copy behind on cgit.kde.org can indeed cause unnecessary work for someone not getting the memo. Just because it did not happen in kraft's case does not mean it can not, and did not happen in other cases. I know of at least one project that got dual porting efforts as a result, and there are many KDE projects that got new developers after being neglected for a very long time. So making sure your repository was only found in one place was a good step, albeit unfortunate for losing translations.
